ui/app/services/version.js

```diff
@@ -3,81 +3,71 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
-import { readOnly, match, not } from '@ember/object/computed';
 import Service, { inject as service } from '@ember/service';
-import { computed } from '@ember/object';
-import { task } from 'ember-concurrency';
+import { keepLatestTask, task } from 'ember-concurrency';
+import { tracked } from '@glimmer/tracking';
 
-const hasFeatureMethod = (context, featureKey) => {
-  const features = context.get('features');
-  if (!features) {
-    return false;
-  }
-  return features.includes(featureKey);
-};
-const hasFeature = (featureKey) => {
-  return computed('features', 'features.[]', function () {
-    return hasFeatureMethod(this, featureKey);
-  });
-};
-export default Service.extend({
-  _features: null,
-  features: readOnly('_features'),
-  version: null,
-  store: service(),
-
-  hasPerfReplication: hasFeature('Performance Replication'),
+export default class VersionService extends Service {
```

Comment:
had to update this to use in a typescript file, I can pull into a separate PR if preferred

Reply:
No problem this works! :)